README.md

```diff
@@ -30,7 +31,7 @@ For reference documentation see our [website](https://www.tensil.ai/docs/).
 
 ```
 docker pull tensilai/tensil
-docker run -v $(pwd):/work -w /work -it tensilai/tensil bash
+docker run -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd):/work -w /work -it tensilai/tensil bash
```

Comment:
what was this needed for?

Reply:
Without this docker will run as root, and so all files it creates are owned by the root. This can be annoying once you are exit the container and need to remove or change these files.